src/main/java/com/softserve/edu/cajillo/config/ModelMapperConfig.java

```diff
@@ -1,28 +1,28 @@
-package com.softserve.edu.cajillo.config;
-
-import org.modelmapper.ModelMapper;
-import org.modelmapper.convention.MatchingStrategies;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
-@Configuration
-public class ModelMapperConfig {
-
-    @Bean
-    public ModelMapper modelMapper() {
-        ModelMapper modelMapper = new ModelMapper();
-        modelMapper.getConfiguration().setMatchingStrategy(MatchingStrategies.STRICT);
-        mapEntitiesToDto(modelMapper);
-        mapDtoToEntities(modelMapper);
-        return modelMapper;
-    }
-
-    private void mapEntitiesToDto(ModelMapper modelMapper) {
-
-    }
-
-    private void mapDtoToEntities(ModelMapper modelMapper) {
-
-    }
-
-}
+package com.softserve.edu.cajillo.config;
+

```

Comment:
Why same part of code ( all classes in this commit) was deleted and created one more time???